### PR TITLE
Make unlit path sample textures with nearest filtering

### PR DESCRIPTION
**Pull request by Noxime**

### CODE REVIEW

File: rend3/shaders/lighting/pixel.glsl [hunk 1]
```diff
@@ -47,7 +47,11 @@ PixelData get_per_pixel_data(MATERIAL_TYPE material) {
 
     if (MATERIAL_FLAG(FLAGS_ALBEDO_ACTIVE)) {
         if (HAS_ALBEDO_TEXTURE) {
-            pixel.albedo = texture(sampler2D(ALBEDO_TEXTURE, linear_sampler), coords);
+            if (MATERIAL_FLAG(FLAGS_UNLIT)) {
```
Comment (cwfitzgerald) [inline rend3/shaders/lighting/pixel.glsl]: On second thought we should just have a new material flag that determines which of the two samplers to use.

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]
Code looks fine, just a change of heart in the shader

**Comment by cwfitzgerald** [comment]
Bump on this?

**Comment by Noxime** [comment]
Oh sorry, I haven't worked at all on this, I've been busy with other stuff, and probably won't get around to this for next few days either. You are free to implement this yourself if you have time :)

**Comment by cwfitzgerald** [comment]
No rush at all, just wanted to make sure you didn't forget :)

**Review by cwfitzgerald — APPROVED** [review]
Thanks so much!

**Pull request merged**